src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/UseConcreteTypeTests.cs

```diff
@@ -11,6 +11,83 @@ namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
 {
     public static partial class UseConcreteTypeTests
     {
+        [Fact]
```

Comment:
Please add `WorkItem("%github_issue_url%")` attribute to each of the added test.

Reply:
Done.